src/main/io/serial_4way.c

```diff
@@ -97,6 +97,8 @@
 #define DAT_TIMEOUT_US  10000
 #define CRC_TIMEOUT_US  10000
 
+#undef TIMEOUT_4WAYIF
```

Comment:
grep can't find it - so we are never be able to enable it

Reply:
```
$ find src -name "*.[ch]" | xargs grep TIMEOUT_4WAYIF
src/main/io/serial_4way.c:#undef TIMEOUT_4WAYIF
src/main/io/serial_4way.c:#ifdef TIMEOUT_4WAYIF
```

Reply:
That's what I meant it is not #defined before - neither in cloud API. So I think this #undef won't be needed?

Reply:
Gotcha. Should it also have a “USE_” prefix?

Reply:
Yes if need externally using custom defines `USE_` is stripped along the way.